site/content/docs/02-template-syntax.md

```diff
@@ -210,17 +210,37 @@ If a *key* expression is provided — which must uniquely identify each list ite
 
 You can freely use destructuring patterns in each blocks.
 
-```html
-{#each items as { id, name, qty }, i (id)}
+```sv
+{#each items as {id, name, qty}, i (id)}
 	<li>{i + 1}: {name} x {qty}</li>
 {/each}
 ```
 
 ---
 
+You can also use rest syntax when destructuring objects *(since [v3.2.0](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#320))*.
+
+```sv
+{#each items as {id, ...rest}}
+	<li><span>{id}</span><MyComponent {...rest} /></li>
+{/each}
+```
+
+---
+
+You can also use rest syntax when destructuring arrays *(since [v3.2.0](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#320))*.
+
+```sv
+{#each items as [id, ...rest]}
+	<li><span>{id}</span><MyComponent {...rest} /></li>
```

Comment:
This doesn't make a whole lot of sense, since we're talking about an array of items, rather than an object with the remaining properties.

Changing this to something like
```html
<MyComponent remaining={rest} />
```
would potentially make sense.

Reply:
oooops, you're right fixed thanks!